model/Software/Properties/isDirectory.md

```diff
@@ -0,0 +1,18 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# isDirectory
+
+## Summary
+
+If true, denotes the Element is a directory.
+
+## Description
+
+If true, denotes the Element is a directory.
+
+## Metadata
+
+- name: isDirectory
+- Nature: DataProperty
+- Range: xsd:boolean
```

Comment:
@kestewart - I think you raise a good issue - but I think it can be separated from the File/Directory issue.  It would be nice to identify if a packageFileName is present or not, but that could be a separate property.  Something we could solve post RC2.